src/main/java/teammates/storage/api/AdminEmailsDb.java

```diff
@@ -225,7 +229,11 @@ public AdminEmailAttributes getAdminEmail(String subject, Date createDate){
         
         Iterator<AdminEmail> it = adminEmailList.iterator();
         while(it.hasNext()){
-            list.add(new AdminEmailAttributes(it.next()));
+            AdminEmail adminEmail = it.next();
+            
+            if (!JDOHelper.isDeleted(adminEmail)) {
```

Comment:
Hmm relevant as in? This check is checking if the items retrieved from database is deleted or not before adding them to the list. I'm not too sure what that check on line 226 actually does tho.